commands/addons/open.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+let opn = require('opn');
+let fs  = require('fs');
+
+function open (url) {
+  cli.log(`Opening ${cli.color.cyan(url)}...`);
+  return new Promise(function (fulfill, reject) {
+    opn(url, {wait: false}, function (err) {
+      if (!err) return fulfill();
+      cli.error('Cannot open page.');
+      cli.error(err);
+      reject(new Error(`Open ${cli.color.cyan(url)} in your browser.`));
+    });
+  });
+}
+
+const ssoPath = '/tmp/heroku-sso.html';
+
+function writeTemplate (ctx, sso, path) {
```

Comment:
Maybe rename this to `writeSudoTemplate`. Threw me for a second because I thought this was used for all kinds of SSO.

In any cause... I really wish we could find a way to put the sudo special case into the `addons-sso` app itself, but I'm not really sure how that's doable with the way it works with identity.